.gitignore

```diff
@@ -5,3 +5,10 @@
 **/__pycache__/
 logs/
 thunder-tests
+
+redis-env/
+*.swp
```

Comment:
this is probably some vim open file, or some other editor maybe using the same, are you sure you want to ignore that? (OK with me, I don't use that)